packages/ui-lib/src/theme/variables/Colors.ts

```diff
@@ -0,0 +1,545 @@
+    --red-a11: #ff9795ef;
+    --red-a10: #ff7272bf;
+    --red-a9: #ff7979ce;
+    --red-a8: #ff7878a1;
+    --red-a7: #ff717178;
+    --red-a6: #fd5f5f5f;
+    --red-a5: #fd464a4d;
+    --red-a4: #ff33383c;
+    --red-a3: #fd383828;
+    --red-a2: #fe48360e;
+    --red-a1: #de000005;
+    --black-1: #000;
+    --black-a12: #000000e8;
+    --black-a11: #00000090;
+    --black-a10: #0000007a;
+    --black-a9: #00000070;
+    --black-a8: #00000038;
+    --black-a7: #00000024;
+    --black-a6: #0000001d;
+    --black-a5: #00000017;
+    --black-a4: #00000012;
+    --black-a3: #0000000c;
+    --black-a2: #00000007;
+    --black-a1: #00000003;
+    --orange-2: #2b1400;
+    --orange-3: #391a03;
+    --orange-4: #441f04;
+    --orange-5: #4f2305;
+    --orange-6: #5f2a06;
+    --orange-7: #763205;
+    --orange-8: #943e00;
+    --orange-9: #f76808;
+    --orange-10: #ff802b;
+    --orange-11: #ff8b3e;
+    --orange-12: #feeadd;
+    --orange-a12: #ffeee1fa;
+    --orange-a11: #ff8c3ffa;
+    --orange-a10: #ff842cfa;
+    --orange-a9: #ff6b08f6;
+    --orange-a8: #fe660085;
+    --orange-a7: #ff640363;
+    --orange-a6: #ff650649;
+    --orange-a5: #fe620137;
+    --orange-a4: #fe61002a;
+    --orange-a3: #fd54001e;
+    --orange-a2: #fd37000e;
+    --orange-a1: #0000;
+    --grey-2: #18191b;
+    --grey-3: #212225;
+    --grey-4: #272a2d;
+    --grey-5: #2e3135;
+    --grey-6: #363a3f;
+    --grey-7: #43484e;
+    --grey-8: #5a6169;
+    --grey-9: #696e77;
+    --grey-10: #777b84;
+    --grey-11: #9a9fa7;
+    --grey-12: #d5d8dc;
+    --grey-a12: #fcfdffef;
+    --grey-a11: #f1f7feb5;
+    --grey-a10: #e5edfd7b;
+    --grey-a9: #dfebfd;
+    --grey-a8: #d9edff5d;
+    --grey-a7: #d9edff40;
+    --grey-a6: #d6ebfd30;
+    --grey-a5: #d9edfe25;
+    --grey-a4: #d3edf81d;
+    --grey-a3: #ddeaf814;
+    --grey-a2: #d8f4f609;
+    --grey-a1: #0000;
+    --white-1: #fff;
+    --white-a12: #ffffffeb;
+    --white-a11: #ffffff97;
+    --white-a10: #ffffff72;
+    --white-a9: #ffffff62;
+    --white-a8: #ffffff3f;
+    --white-a7: #ffffff2d;
+    --white-a6: #ffffff20;
+    --white-a5: #ffffff16;
+    --white-a4: #ffffff0e;
+    --white-a3: #ffffff09;
+    --white-a2: #ffffff03;
+    --white-a1: #fff0;
+
+      /* feedback */
+    --error: var(--warning-8);
+    --info: var(--primary-7);
+    --neutral: var(--grey-9);
+    --success: var(--success-8);
+    --warning: var(--caution-8);
+
+      /* icons */
+    --inverse: var(--white-1);
+    --mono: var(--grey-12);
+    --dimmed: var(--grey-11);
+    --subtle: var(--grey-10);
+    --primary: var(--primary-9);
+    --danger: var(--warning-9);
+    --white: var(--white-1);
+
+    --spinner-danger: var(--warning-8);
+    --spinner-secondary: var(--grey-8);
+    --spinner-primary: var(--primary-6);
+    --spinner-simple: var(--grey-a8);
+
+    /* --main-background-gradient: linear-gradient(180.00deg, var(--grey-2) 0%, var(--grey-3) 100%);
+    --main-background-color: var(--grey-3); */
+  }
```

Comment:
Is the main background variable here not longer needed?

Reply:
I think it should be here. I've uncommented it and in the next PR, this should be moved anyway.